### ISSUE TITLE
Add cors support to s3_bucket module

### ISSUE THREAD

**Issue by bermudezjd** [ansible/ansible-modules-extras#1163]
##### ISSUE TYPE

Feature Request
##### COMPONENT NAME

s3_bucket
##### ANSIBLE VERSION

N/A
##### SUMMARY

Add cors support to s3_bucket module
@wimnat : mind if I take a crack at adding cors xml support in my fork for an eventual PR?

**Comment by wimnat** [comment]
Hi @bermudezjd.  I did do some work on cors support.  I'll see if I can pull it out.

I wouldn't add it to s3_bucket.  I think my plan was a new module s3_cors_rule which added and removed individual rules as necessary.  I think this is cleaner that trying to add all your cors rules in one module.

PS you might want to move this conversation to ansible devel google group.

**Comment by wimnat** [comment]
Linking PR https://github.com/ansible/ansible-modules-extras/pull/1177

**Comment by ansibot** [comment]
@wimnat, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@bermudezjd ping, this issue is still waiting on your feedback. We will close the issue if you do not respond.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@bermudezjd ping, this issue is still waiting on your feedback. We will close the issue if you do not respond.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by bermudezjd** [comment]
Not sure if it's worth reconfiguring this PR.  I've already done so 3 or 4 times and it never seems to get approved.  Any reason why I should put this out again?

**Comment by ansibot** [comment]
This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by ansibot** [comment]
This issue was migrated to https://github.com/ansible/ansible/issues/29238
